semver/range_test.ts

```diff
@@ -185,28 +183,11 @@ Deno.test({
   },
 });
 
-Deno.test("unlockedPrereleaseRange", function () {
-  const versions: [string, string][] = [
-    ["*", "1.0.0-rc1"],
-    ["^1.0.0-0", "1.0.1-rc1"],
-    ["^1.0.0-rc2", "1.0.1-rc1"],
-    ["^1.0.0", "1.0.1-rc1"],
-    ["^1.0.0", "1.1.0-rc1"],
```

Comment:
These cases don't pass with `npm:semver`. For example:

```js
import * as semver from "npm:semver";
const range = new semver.Range("*");
console.log(range.test("1.0.0-rc"));
```

The above prints false, but expected true here.

Reply:
Ditto. Do we have this `npm:server` test somewhere?

Reply:
Added these cases back to `testRange() returns false when the version is not in the range` with fixed expectations